src/contract.rs

```diff
@@ -27,14 +29,32 @@ pub fn init<S: Storage, A: Api, Q: Querier>(
     env: Env,
     msg: InitMsg,
 ) -> StdResult<InitResponse> {
+    // Check name, symbol, decimals
+    if !is_valid_name(&msg.name) {
+        return Err(StdError::generic_err(
+            "Name is not in the expected format (3-30 UTF-8 bytes)",
+        ));
+    }
+    if !is_valid_symbol(&msg.symbol) {
+        return Err(StdError::generic_err(
+            "Ticker symbol is not in expected format [A-Z]{3,6}",
+        ));
+    }
```

Comment:
Why are we still enforcing this?

Reply:
as discussed in the call, it's just a basic default, people the need to change this can fork the repo.